src/renderer/src/components/song/song-queue/SongQueue.tsx

```diff
@@ -105,14 +108,31 @@ const SongQueue: Component = () => {
               draggable={true}
               onSelect={() => window.api.request("queue::play", s.path)}
               onDrop={onDrop(s)}
-            >
-              <SongContextMenuItem onClick={() => window.api.request("queue::removeSong", s.path)}>
-                Remove from queue
-              </SongContextMenuItem>
-            </SongItem>
+              showSignal={showSignal}
+              setSong={setSong}
+            ></SongItem>
           )}
         />
       </div>
+      <SongContextMenu show={showSignal}>
+        <SongContextMenuItem
+          onClick={() => {
+            console.log("todo");
+          }}
+        >
+          <p>Add to playlist</p>
+          <i class="ri-add-line"></i>
+        </SongContextMenuItem>
+
+        <SongContextMenuItem
+          onClick={() => {
+            window.api.request("queue::removeSong", song()?.path);
+          }}
+        >
+          <p>Remove from queue</p>
+          <i class="ri-delete-back-2-line"></i>
```

Comment:
And here.

Reply:
Done. Replaced with `<PlusIcon />` and `<DeleteIcon />`